docs/website/docs/guides/deployment-configurations/bare-metal.md

```diff
@@ -5,7 +5,7 @@ tags:
   - CPU
 ---
 
-# Run on a Bare-Metal Platform
+# Running on a Bare-Metal Platform
 
 IREE supports CPU model execution on bare-metal platforms. That is, platforms
```

Comment:
s/That is, platforms without/Bare metal platforms have no operating system support/?

(The current wording does form a sentence grammar-wise.)